shared/macros-bash.jinja

```diff
@@ -1677,14 +1677,17 @@ fi
     Remediation for grub2 bootloader arguments
 #}}
 {{% macro grub2_bootloader_argument_remediation(ARG_NAME, ARG_NAME_VALUE) %}}
+{{% set grub_helper_args = ["--update-kernel=ALL", "--args=" ~ ARG_NAME_VALUE] -%}}
```

Comment:
```suggestion
{{% set grub_helper_args = ["--update-kernel=ALL", "--args= {{{ ARG_NAME_VALUE }}}"] -%}}
```
This might be help clear up that is going on here.